src/engine/renderer/Material.cpp

```diff
@@ -1652,6 +1652,7 @@ void MaterialSystem::CullSurfaces() {
 		}
 		matrix_t viewMatrix;
 		if ( PVSLocked ) {
+			VectorClear( origin );
```

Comment:
This can now be removed, the fix is on `master`.

Reply:
Now removed. Let's see if the CI is now happy with it and I'll merge this.